src/poetry/console/commands/add.py

```diff
@@ -285,3 +293,7 @@ def notify_about_existing_packages(self, existing_packages: list[str]) -> None:
         for name in existing_packages:
             self.line(f"  • <c1>{name}</c1>")
         self.line(self._hint_update_packages)
+
+    def migrate_dev_dependencies(self, poetry_content: dict[str, Any]) -> None:
```

Comment:
This is now what I had in mind earlier. However, this method shouldn't be exported -- so it should either be `_migrate_dev_dependencies`, or just inlined (which I think is better given how trivial it is).